cli.js

```diff
@@ -127,9 +128,14 @@ function setOptionDefaults(module) {
 function setCommandDefaults(module) {
   let self = {};
 
+  if (module.usage) {
+    module.usage = module.usage.replace(/\$0/g, CLI_NAME);
+  }
+
   _.defaults(self, module, {
     options: {},
     describe: `${module.command} command`,
+    usage: `Usage: ${CLI_NAME} ${module.command}${ _.keys(module.options).length > 0 ? ' [options]' : ''}`,
```

Comment:
I think I might simplify to just say `[options]` - it's understood that if a command takes no options, then none are needed.